pages/common/sngrep.md

```diff
@@ -0,0 +1,20 @@
+# sngrep
+
+> Tool for displaying SIP calls message flows from terminal.
+> More information: <https://github.com/irontec/sngrep>.
+
+- Visualize SIP packets from a pcap file:
```

Comment:
```suggestion
- Visualize SIP packets from a `pcap` file:
```

Reply:
In the past, we had file extensions in example descriptions all caps, e.g. here: https://github.com/tldr-pages/tldr/blob/main/pages/common/ybmtopbm.md 

```suggestion
- Visualize SIP packets from a PCAP file:
```